lib/matplotlib/tests/test_path.py

```diff
@@ -273,6 +273,82 @@ def test_path_deepcopy():
     copy.deepcopy(path2)
 
 
+def test_path_intersect_path():
+    # test for the range of intersection angles
+    for phi in np.linspace(0, 2*np.pi, 180):
```

Comment:
Going through 180 angles seems a bit much. Have you checked, that the test still runs fast (<< 1s)? Otherwise it should be sufficient to go through some selected angles like
~~~
angles = np.array([0, 30, 45, 90, ...])
angles = np.concatenate([angles, angles+1, angles-1])
~~~

Reply:
@timhoffm, it does take about 1s to complete. I will use the approach you've suggested.

Reply:
So, I've reduced the total range of angles to 30 so it runs faster.